### PR TITLE
scripts to run the param finder, and to freeze it with cx_freeze.

### PR DESCRIPTION
**Pull request by marionleborgne**
cc @ywcui1990

### CODE REVIEW

File: htmresearch/frameworks/utils/param_finder.py [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+  output = np.zeros([len(widths), len(data)])
+  for ind, width in enumerate(widths):
+    wavelet_data = wavelet(min(10 * width, len(data)), width)
+    output[ind, :] = _convolve(data, wavelet_data,
+                               mode='same')
+  return output
+
+
+
+def read_csv_files(fileName):
```
Comment (ywcui1990) [inline htmresearch/frameworks/utils/param_finder.py]: This function (and the csv module) is only needed for loading the example data. We may not need to include it in the param_finder

File: requirements.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@
 pandas
 plotly
```
Comment (ywcui1990) [inline requirements.txt]: why is plotly required?

**Comment by ywcui1990** [comment]
:+1:

File: requirements.txt (hunk 2 above)
Comment (marionleborgne) [inline requirements.txt]: I don't know, it was there before

File: htmresearch/frameworks/utils/param_finder.py (hunk 1 above)
Comment (marionleborgne) [inline htmresearch/frameworks/utils/param_finder.py]: We may or may not. The param finder will need to read data from the file system in Unicorn. Also, the actual script in unicorn will most likely be structured differently / moved somewhere else, so it's probably ok to leave this script like this for now. What are your thoughts?

**Pull request merged**